src/message.ts

```diff
@@ -441,20 +441,21 @@ export class Message implements Sayable {
       return contactList
     }
 
-    // define magic code `8197` to identify @xxx
+    // define magic code `8197` to identify @xxx (wechat version > 6.5, old version use normal blank `32`)
     const AT_SEPRATOR = String.fromCharCode(8197)
```

Comment:
It is really rare that user use wechat below 6.5. So this version I decide to ignore this situation